README.md

```diff
@@ -59,10 +60,11 @@ See the file [COPYING](COPYING) for more details.
 
 HISTORY
 -------
+XQF was originally written by Roman Pozlevich from Pereslavl-Zalessky, Russia. Over 17 years of existence the project has had the following major developers:
```

Comment:
Specifying homeplace of each developer is important if our names are more than just 'author' line in git log :). Appending (like Ludwig Nussel, town_name, Germany) should be enough